setup.cfg

```diff
@@ -1,7 +1,7 @@
 [wipac:cicd_setup_builder]
 pypi_name = wipac-mqclient-pulsar
 python_min = 3.6
-python_max = 3.9
+python_max = 3.10
```

Comment:
you don't need to define a max here, it'll default to 3.10

Reply:
I define a max because when py3.11 is released, pulsar will again break for a few months until they upgrade their c++ pybindings.  There are a number of new changes they might have to fix.